sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_models.py

```diff
@@ -3011,7 +3011,7 @@ class DocumentBarcode:
      "MicroQRCode", "Aztec", "DataMatrix", "MaxiCode"."""
     value: str
     """Barcode value."""
-    polygon: Sequence[Point]
+    polygon: Optional[Sequence[Point]]
```

Comment:
Don't we return an empty list? Should this really be optional?

Reply:
Optional means "can be None". If we return it as an empty list, we should leave it as non-optional